process/accept-orders.py

```diff
@@ -341,7 +349,7 @@ def accept(game, locale, stream, extend=None):
 locale = sys.argv[2]
 infile = sys.stdin
 if len(sys.argv)>3:
-    infile = open(sys.argv[3], "r")
+    infile = open(sys.argv[3], "rb")
```

Comment:
warum hier nicht io.open? Was ist der Unterschied zwischen dem built-in open() in Python 3 und io.open?

Reply:
Good catch. In python3 ist es dasselbe. In python2 nicht, aber frag mich nicht, was der Unterschied ist, jedenfalls wird io.open empfohlen, wenn es für beide Versionen kompatibel sein soll.